pgjdbc/src/test/java/org/postgresql/test/jdbc42/GetObject310BinaryTest.java

```diff
@@ -9,10 +9,6 @@
 
 public class GetObject310BinaryTest extends GetObject310Test {
 
-  public GetObject310BinaryTest(String name) {
```

Comment:
Do GetObject310Test and GetObject310BinaryTest work after your changes?

Previously GetObject310Test was powered by `extends BaseTest`, and now you seem to convert it to JUnit4 format.
That means updateProperties will never be called. Will it?